sstables/sstables.cc

```diff
@@ -3178,6 +3178,9 @@ future<> init_metrics() {
 
         sm::make_gauge("bloom_filter_memory_size", [] { return utils::filter::bloom_filter::get_shard_stats().memory_size; },
             sm::description("Bloom filter memory usage in bytes.")),
+
+        sm::make_counter("malformed_sstable_errors", [] { return sstables_stats::get_shard_stats().malformed_sstable_exceptions; },
```

Comment:
It needs the skip_when_empty flag.
Add total as a suffix to the name.
Do we need this at a shards level? Maybe we can have one per node. If that's the case, add aggregate by shard